src/content/docs/query-your-data/explore-query-data/use-charts/on-demand-forecasting.mdx

```diff
@@ -0,0 +1,129 @@
+---
+title: On-demand forecasting
+tags:
+  - Forecast on a chart
+redirects:
+freshnessValidatedDate: never
+---
+<Callout title="preview">
+  We're still working on this feature, but we'd love for you to try it out!
+
+  This feature is currently provided as part of a preview program pursuant to our [pre-release policies](/docs/licenses/license-information/referenced-policies/new-relic-pre-release-policy).
+</Callout>
+
+
+Forecasting in New Relic involves using historical data to predict future trends and potential performance issues. 
+By analyzing historical data, forecasting provides insights in to how these metrics might behave in the future. 
+This proactive approach helps you to anticipate issues before they occur, enabling timely interventions to maintain optimal system performance. 
+Forecasting is a powerful tool for decision-making, helping teams to allocate resources effectively, and plan for potential challenges.
+
+<Callout variant="tip">
+  New Relic currently uses the Holt-Winters model for forecasting, which reliably captures seasonal patterns and trends in your data.
```

Comment:
It'd be nice to have an explanatory link for Holt-Winters, maybe this one: https://en.wikipedia.org/wiki/Exponential_smoothing#Triple_exponential_smoothing_(Holt_Winters)